guides/common/modules/proc_creating-an-activation-key.adoc

```diff
@@ -11,32 +11,17 @@ endif::[]
 Subscription Manager attaches subscriptions differently depending on the following factors:
 
 * Are there any subscriptions associated with the activation key?
-* Is the auto-attach option enabled?
 * For {RHEL} 8 hosts: Is there system purpose set on the activation key?
 
 Note that {Project} automatically attaches subscriptions only for the products installed on a host.
-For subscriptions that do not list products installed on {RHEL} by default, such as the Extended Update Support (EUS) subscription, use an activation key specifying the required subscriptions and with the auto-attach disabled.
+For subscriptions that do not list products installed on {RHEL} by default, such as the Extended Update Support (EUS) subscription, use an activation key specifying the required subscriptions.
 
-Based on the previous factors, there are three possible scenarios for subscribing with activation keys:
+Based on the previous factors, there is one possible scenario for subscribing with activation keys:
 
-. Activation key that attaches subscriptions automatically.
+* Activation key that attaches subscriptions automatically.
 +
-With no subscriptions specified and auto-attach enabled, hosts using the activation key search for the best fitting subscription from the ones provided by the Content View associated with the activation key.
-This is similar to entering the `subscription-manager --auto-attach` command.
+With no subscriptions specified, hosts using the activation key search for the best fitting subscription from the ones provided by the Content View associated with the activation key.
 For {RHEL} 8 hosts, you can configure the activation key to set system purpose on hosts during registration to enhance the automatic subscriptions attachment.
```

Comment:
You can still set system purpose values on an activation key, but the reason is different. Also it's not just for RHEL 8.
```suggestion
You can configure the activation key to set system purpose attributes on hosts during registration. This can help determine which repositories are available on the host, and also helps with reporting in the Subscriptions service of the Red Hat Hybrid Cloud Console.
```